CHANGELOG.md

```diff
@@ -25,6 +25,12 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - The semantic conventions used by `go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp/example` are upgraded to v1.20.0. (#4320)
 - The semantic conventions used by `go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp`are upgraded to v1.20.0. (#4320)
 - Updated configuration schema to include `schema_url` for resource definition and `without_type_suffix` and `without_units` for the Prometheus exporter. (#4727)
+- The semantic conventions used by `go.opentelemetry.io/contrib/detectors/aws/ecs` are upgraded to v1.24.0. (#4803)
+- The semantic conventions used by `go.opentelemetry.io/contrib/detectors/aws/lambda` are upgraded to v1.24.0. (#4803)
```

Comment:
```suggestion
- The semantic conventions used by `go.opentelemetry.io/contrib/detectors/aws/lambda` resource detector are upgraded to v1.24.0. (#4803)
```

OR

https://github.com/open-telemetry/opentelemetry-go-contrib/pull/4803#pullrequestreview-1819478769

Personally I am leaning more towards adding `resource detector` to all changelog entries.